pyface/ui/qt4/tests/test_gui.py

```diff
@@ -35,7 +35,7 @@ class SimpleApplication(HasStrictTraits):
     application_running = Event()
 
     def __init__(self):
-        super(HasStrictTraits, self).__init__()
```

Comment:
Just pointing out this also differs from the rest in that it did not call `super(SimpleApplication, self).__init__()` previously